benchmarks/Elastic.Transport.Benchmarks/Elastic.Transport.Benchmarks.csproj

```
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

```suggestion
    <OutputType>Exe</OutputType>
    <IsPackable>false</IsPackable>
```

Since this repository is set up to publish to nuget on tagging lets get this one in to prevent it accidentally winding up there :)

---

Done!